pkg/virt-handler/vm.go

```diff
@@ -20,7 +20,6 @@
 package virthandler
```

Comment:
@andreyod would you add the motivation for this change into the commit message? 
Also, I would prefer if the second commit is squashed into the first one, for future readability og `git log`.

Reply:
I see that you say
>  Reducing direct API calls should improve the system performance and scalability.
in the PR text; still, stating this in the commit message, too, would be helpful.

Reply:
Done

Reply:
Thanks. Though egotistically, I very much prefer more verbose explanation. I prefer following https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines